### ISSUE TITLE
alert instantly closes after being opened when firing it on touchstart

### ISSUE THREAD

**Issue by chrisSowerby** [t4t5/sweetalert#619]
On chrome mobile browser and desktop chrome (emulator) the alert instantly closes after being opened. This only happens when touchstart is used as the event.  on click works fine.

iphone safari handles the touchstart fine.
I have checked to see if touchstart is firing twice as i though this might be why, but it is defiantly only firing once on chrome. I just dont know what is triggering the instant close of the alert.

```
                $(document).on("touchstart", "img", function() {

                        swal({
                            title: "Search",
                            text: "Please type your search query:",
                            type: "input",
                            showCancelButton: true,
                            closeOnConfirm: false,
                            showLoaderOnConfirm: true,
                            animation: "slide-from-top",
                            inputPlaceholder: "",
                            allowEscapeKey: true
                        }, function(inputValue) {
                            if (inputValue === false) {
                                return false;
                            }
                            if (inputValue === "") {
                                swal.showInputError("Please type your search query into the field above.");
                                return false;
                            }
                            // do something 
                        });

                });
```

Regards.

**Comment by raymondsteers** [comment]
wrap it in a 100ms settimeout and it wont do that anymore. Same problem if fired from a touchend
`setTimeout(function(){
		swal({title: "Because",
		    text: "Touch events are wonky"
		    });
	    }, 100);`

**Comment by t4t5** [comment]
Just tested this in the Chrome emulator and it seems like this has been fixed in the 2.0 release. :)
